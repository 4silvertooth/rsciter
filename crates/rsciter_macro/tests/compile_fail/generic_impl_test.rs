struct S;
#[rsciter_macro::xmod]
impl<T> S {
}

fn main() {}