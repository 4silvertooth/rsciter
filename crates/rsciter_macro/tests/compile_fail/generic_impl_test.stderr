error: #[rsciter::xmod] Generic impl blocks are not supported!
 --> tests/compile_fail/generic_impl_test.rs:3:5
  |
3 | impl<T> S {
  |     ^
